li-utils/src/main/pegasus/com/linkedin/common/DashboardUrn.pdl

```diff
@@ -21,10 +21,10 @@ namespace com.linkedin.common
   },
   {
     "name" : "dashboardId",
-    "doc" : "Unique id of the dashboard assigned by the specific dashboard tool",
+    "doc" : "Unique id for the dashboard. This id should be globally unique for a dashboarding tool even when there are multiple deployments of it. As an example, dashboard URL could be used here for Looker such as 'looker.linkedin.com/dashboards/1234'",
     "type" : "string",
-    "maxLength" : 20
+    "maxLength" : 200
   }],
-  "maxLength" : 36
+  "maxLength" : 239
```

Comment:
What is the purpose of this maxLength value? 
Should the maxLength be (36 + 200-20) = 216 if you are trying to change 20 -> 200 and updating the global maxLength?

Reply:
First of all, I'm sure you know that these (xxxurn.pdl) are auto-generated models at LinkedIn. URN generator creates these models (and calculates the total maxLength) and they're not checked in anywhere for internal URNs. Because we don't have urn generator in OSS, we're just creating these models manually and checkin to the source code.

That being said, I was just manually calculating and updating those numbers as a best effort. I assume most of those numbers in other urn models are just plain wrong (including previous version of this). So, how I found this number is just:
len("urn:li:dashboard:(,)") + maxlen(dashboardTool) + maxlen(dashboardId) = 239

Reply:
It was 240 actually, fixed it :)